src/Symfony/Component/Mime/Tests/NamedAddressTest.php

```diff
@@ -24,4 +24,19 @@ public function testConstructor()
         $this->assertEquals('Fabien <fabien@xn--symfon-nwa.com>', $a->toString());
         $this->assertEquals('fabien@xn--symfon-nwa.com', $a->getEncodedAddress());
     }
+
+    public function nameEmptyDataProvider(): array
+    {
+        return [[''], [' '], [" \r\n "]];
+    }
+
+    /**
+     * @dataProvider nameEmptyDataProvider
+     */
+    public function testNameEmpty(string $name)
+    {
+        $mail = 'mail@example.org';
+
+        $this->assertSame($mail, (new NamedAddress($mail, $name))->getEncodedNamedAddress());
```

Comment:
Wait, this must be `Address`, not `NamedAddress`

Reply:
And the File `NamedAddress` is also wrong, isnt it?

Reply:
This PR is for 4.3, so this is good. I'm doing the change for 4.4.